test/fixtures/globalid/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>globalid</title>
+  <meta name="description" content="test that globalid tracking works"></meta>
+  <script src="/dist/td.js"></script>
+</head>
+<body>
+
+<script type="text/javascript">
+  (function () {
+    function setBackground (color) {
+      document.body.style.background = color
+    }
+
+    function setStatus (text) {
+      document.getElementById('status').innerHTML = text
+    }
+
+    var td = new Treasure({
+      database: 'test_db_request',
+      writeKey: '91/96da3cfb876cc50724d0dddef670d95eea2a0018',
+      host: 'in.treasuredata.com'
+    })
+
+    td.fetchGlobalID(function (value) {
```

Comment:
I think it'd be a better test to call this twice and make sure the result matches.